front/src/form/bsdasri/components/grouping/BsdasriTableGrouping.tsx

```diff
@@ -113,7 +113,8 @@ export default function BsdasriTableGrouping({
     const aggregatedPackagings = aggregatePackagings(packagings);
 
     setFieldValue("emitter.emission.packagings", aggregatedPackagings);
-  }, [selectedItems, data, setFieldValue, values]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [selectedItems, data, setFieldValue]);
```

Comment:
Au lieu d'ignorer l'erreur on pourrait peut être:
- se défaire complètement du `useEffect` (cf https://react.dev/learn/you-might-not-need-an-effect)
- ou retirer le `values` de la computation. Peut être qu'on peut définir `emitter.emission.weight.isEstimate` sans dépendre de la valeur courante. Après tout on défini nous même la quantité. On pourrait imaginer dans le reduce ligne 99 un check pour savoir si on fallback sur 0 ou si on a toujours une valeur, et ajuster le `isEstimate` en fonction. Sinon toujours mettre à false, je sais pas à voir

Reply:
ok j'avoue que je suis pas trop rentré dans le détail, je vais voir ce que je peux faire

Reply:
J'ai modif (voir commit suivant) en gardant le useEffect, mais en définissant le isEstimate en fonction du calcul de weight (si un des weight est manquant ou 0, partir du principe que le total est une estimation). Je suppose que c'est ce que tu avais en tête pour l'option 2 de ton commentaire.